Tests/Unit/Action/Tool/Message/OidcInitiationActionTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
```

Comment:
All open-sourced files must have a GPL license header.
Could you please set your PhpStorm's live template to add it for every new file?

https://oat-sa.atlassian.net/l/c/F0q7PJq5

Reply:
Added header